backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -66,9 +67,56 @@ public UserInfo createUser(
   public UserInfo createUserInCurrentOrg(
       String username, PersonName name, Role role, boolean active) {
     Organization org = _orgService.getCurrentOrganization();
+
+    Optional<ApiUser> found = _apiUserRepo.findByLoginEmailIncludeArchived(username);
+    if (found.isPresent()) {
+      return reprovisionUserHelper(found.get(), name, org, role);
```

Comment:
nit: in this class there is no existing `reprovisionUser` method, so this doesn't need to be marked as `Helper`